app/assets/javascripts/components/sample_transfer.js.jsx

```diff
@@ -132,46 +103,51 @@ var SampleTransferModal = React.createClass({
     this.setState({
       includeQcInfo: !oldValue
     });
+
+    if ($("#include-qc-check").is(":checked")) {
+      $("#include-qc-check").attr("value", "true");
+    } else {
+      $("#include-qc-check").attr("value", "false");
+    }
```

Comment:
This could optimized & simplified: 
```js
var qcChecked = $("#include-qc-check");
qcChecked.attr("value", !qcChecked.is(":checked");
```

Reply:
Why is this even necessary? Shouldn't the value already be set accordingly?

Reply:
If the checkbox is checked the true value is sent by the form, but if it is not checked, no information is sent for this input by the form, that's why I added it.